astrovascpy/utils.py

```diff
@@ -20,7 +20,7 @@
 import numpy as np
 import pandas as pd
 import psutil
-import scipy as sp
+import scipy as scp
```

Comment:
You save typing 2 letters here. Is that really worth it?

Reply:
let the man develops @matz-e ! Come on, this is still a draft :)

Reply:
LOL. 
I just wanted something different from this:
https://github.com/BlueBrain/AstroVascPy/blob/b2519835e2168ae37bb44154852b393270c59e03/astrovascpy/bloodflow.py#L23
but yes, we can also the full name.

Reply:
LOL.  I just wanted something different from this:
https://github.com/BlueBrain/AstroVascPy/blob/b2519835e2168ae37bb44154852b393270c59e03/astrovascpy/bloodflow.py#L23
but yes, we can also the full name.

Reply:
> let the man develops @matz-e ! Come on, this is still a draft :)

I get a notification, it's fair game ;)